src/Uno.UWP/Generated/3.0.0.0/Windows.ApplicationModel.Contacts/Contact.cs

```diff
@@ -20,8 +20,8 @@ public  partial class Contact
 				global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.ApplicationModel.Contacts.Contact", "IRandomAccessStreamReference Contact.Thumbnail");
 			}
 		}
-		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+#endif
```

Comment:
Don't change the indentation if unnecessary

Reply:
VStudio changes indentation of `#if` / `#endif` lines automagically, at least on my installation (16.4.2).